homeassistant/components/philips_js/media_player.py

```diff
@@ -313,10 +313,11 @@ def update(self):
         self._tv.update()
 
         self._sources = {
-            srcid: source["name"] or f"Source {srcid}"
+            srcid: source.get("name", f"Source {srcid}")
```

Comment:
We're now no longer guarding against the value being falsy, just guarding against a missing key. Is the former case not present?

Reply:
I think the original code was wrong. But maybe i should guard for that too, doesn't hurt.